teacher.rb

```diff
@@ -0,0 +1,12 @@
+class Teacher < Person
+  def initialize(age, name = 'unknown', parent_permission: ture, specialization: 'Physics')
+    super(age, name, parent_permission)
+    @specialization = specialization
+  end
+
+  attr_reader :specialization
```

Comment:
Consider moving this to the top in the class. Remember, to follow Ruby's best practice.

Reply:
Implemented 👍